horizon_autotests/app/ui/table.py

```diff
@@ -54,6 +54,14 @@ def wait_for_status(self, status, timeout=EVENT_TIMEOUT):
                  timeout_seconds=EVENT_TIMEOUT, sleep_seconds=0.1)
             assert cell.value == status
 
+    @pom.timeit
+    def check(self):
```

Comment:
@gdyuldin, that is cool idea to specify method to select row. But actually, as I see, there are some architectural defects in your implementation:
- You don't care whether it was selected already or not.
- You click table cell, but no checkbox. Now it is a happy coincidence that the checkbox is in the first cell in the center. But later it may be changed. So let to provide scalability, when it's not expensive.
- You create method to select, so let to create method to unselect.

And I would prefer `select` and `unselect` method names to ensure uniformity, because `checkbox` element has such methods.

Our target to provide not only tests coverage, but also architectural integrity and completeness for quick tests expansion.

So my variant looks like:

```python
@ui.register_ui(
    checkbox=_ui.CheckBox(By.CSS_SELECTOR, 'input[type="checkbox"]'))
class Row(ui.Row):

    @pom.timeit
    def select(self):
        self.checkbox.select()  # checkbox observes whether it is selected already or not

    @pom.timeit
    def unselect(self):
        self.checkbox.unselect()
```